neqo-transport/src/connection.rs

```diff
@@ -122,21 +124,27 @@ impl Path {
 
 #[derive(Clone, Debug, PartialEq)]
 /// Type returned from process() and process_output(). Users are required to
-/// call these repeatedly until Callback is returned.
+/// call these repeatedly until `Callback` or `None` is returned.
```

Comment:
This implies that these are terminal states, but after None you might get more packets (you have to check the state to tell whether you need to hang around for those).  And after Callback you have to wait and then call back.

Reply:
Took another stab at this comment, lmkwyt.